### DISCUSSION TITLE
Rewards Theater Couldn't Load Rewards

### DISCUSSION THREAD

**Discussion by Rykocalion** [gottagofaster236/RewardsTheater#10] [Q&A]
Hello, Rewards Theater has seem to stop working for me. I even updated to the latest version. However it gives me the error of "Couldn't load rewards. Are you connected to the internet?"  Obviously I am connected and can start stream but still doesn't work. I did log in an authenticated twitch which it says its successful, however I noticed back in obs it still says log in instead of log out after authentication. I am not sure what I can do to fix this.

**Comment by gottagofaster236** [comment]
Thanks for the report, I just reproduced the issue. I'm gonna try to fix it ASAP

**Reply by TMAD413** [reply]
Started browsing for issue reports and landed here...seems I found the answer... Thanks for the response & dedication to fix it.

**Comment by gottagofaster236** [comment]
This is fixed in 1.0.8, please update your plugin @Rykocalion @TMAD413

**Comment by gottagofaster236** [comment]
https://github.com/gottagofaster236/RewardsTheater/releases/tag/1.0.8
